src/bepasty/utils/upload.py

```diff
@@ -92,6 +105,12 @@ def meta_new(cls, item, input_size, input_filename, input_type,
 
     @classmethod
     def meta_complete(cls, item, file_hash):
+        # update TYPE by python-magic if not decided yet
+        if item.meta.get(TYPE_HINT):
+            del item.meta[TYPE_HINT]
```

Comment:
similar here. you could also use `.pop(KEY, False)`

Reply:
fixed